SparkPost/support-docs
# WE-341 Upgrade to Next.js 12

**What Changed**
- Upgrades to Next.js version 12

**How to verify**
- Verify all checks pass


See: 
- https://github.com/netlify/netlify-plugin-nextjs/blob/main/docs/large-functions.md
- https://github.com/netlify/netlify-plugin-nextjs/discussions/706#discussioncomment-1503005
